src/github_review_prompts/tests/test_github_api_integration.py

```
@@ -0,0 +1,561 @@
+"""
+GitHub API統合テスト
+GitHubクライアントのAPI呼び出し、認証、エラーハンドリングを包括的にテスト
+"""
+
+import pytest
+import json
+import sys
+import os
+from unittest.mock import Mock, patch, MagicMock
```

Ruff (0.12.2)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)